src/reports/package/root/package.json

```diff
@@ -12,7 +12,7 @@
         "classnames": "^2.2.6",
         "lodash": "^4.17.15",
         "moment": "^2.24.0",
-        "office-ui-fabric-react": "6.211.0",
+        "office-ui-fabric-react": "7.76.2",
```

Comment:
I'm not 100% aware of the implications of this change. I update this because we have a test to make sure this versions (on the report) are the same versions as in AI-Web (which make sense to me). My main concern here is what does this means in terms of releasing the report package.